Makefile

```diff
@@ -2,7 +2,7 @@ SHELL=/bin/bash
 EXEC=`test -d .stack-work/ && echo "stack exec --" || echo "cabal exec --"`
 HLINT=$(EXEC) hlint
 AULA_SOURCES=-isrc -itests -idist/build/autogen
-FULL_SOURCES=$(AULA_SOURCES) -i$(THENTOS_ROOT_PATH)/thentos-frontend-session/src/ $(THENTOS_ROOT_PATH)/thentos-frontend-session/tests/
+FULL_SOURCES=$(AULA_SOURCES) -i$(THENTOS_ROOT_PATH)/thentos-frontend-session/src/
```

Comment:
why?

(it's not really that important because i am about to release thentos-frontend-session on hackage and this will go away anyway, but this is one of those changes that could use more documentation.)